README.md

```diff
@@ -139,6 +139,65 @@ Note: The version of the ngraph-tensorflow-bridge is not going to be exactly
 the same as when you build from source. This is due to delay in the source 
 release and publishing the corresponding Python wheel. 
 
+### Build and run nGraph in Docker
```

Comment:
Can we move this to another README for docker build. You can have a link to that in this README

Reply:
👍 I've done this in PR #355 . Please switch over to that PR.